integration/test-integration/src/test/resources/apiYaml/operation_level_ratelimit_api.yaml

```diff
@@ -83,15 +83,17 @@ data:
         request: []
         response: []
         fault: []
+    # By placing this POST operation before the GET /pet/{petId} covers the applying of
+    # UNLIMITED throttling policy in combination with other throttling policies.
```

Comment:
Does the order of the operations in API yaml matter?

Reply:
Order changed to check errors after parsing unlimited rl operations.